frontend/src/components/FacilityBooking/ViewBlock.tsx

```diff
@@ -5,6 +5,29 @@ import { scrollToView } from './CurrentTimeLine'
 
 import { StyledViewBooking } from './BlockStyles.styled'
 
+/**
+ *
+ * @param entry (type `number`)
+ * @param scrollTo (type `boolean`, optional)
+ * @param openViewBookingModal: (type `function`)
+ * @param setViewBookingEntryId (type `function`)
+ *
+ * @returns An event block that contains the component displaying the:
+ *          1: Facility
+ *          2: CCA that made the booking
+ *          3: Event name
+ *          4: Date/Time of booking
```

Comment:
Add backslashes for the comments to appear on new lines

```suggestion
 * @returns An event block that contains the component displaying the: \
 *          1: Facility \
 *          2: CCA that made the booking \
 *          3: Event name \
 *          4: Date/Time of booking
```